docs/3. Product Features/03. Flows/2. Flow Actions/10. Save a result for this flow.md

```
@@ -0,0 +1,256 @@
+<br>
+<hr>
+
+
+
+## How to Create a Save Flow Result  in Glific:
+ 1. `Add a New Step`:
+
+      -    In your flow, add a new node after collecting user input.
+
+
+2. `Select "Save Flow Result`:
```

markdownlint-cli2 (0.17.2)

Inconsistent indentation for list items at the same level
Expected: 1; Actual: 0

(MD005, list-indent)